user/job-lifecycle.md

```diff
@@ -9,19 +9,27 @@ Travis CI provides a default build environment and a default set of phases for e
 
 Keep reading to see how you can customize any phase in this process, via your `.travis.yml`.
 
+## The build
+
+The `.travis.yml` file describes the build (process. A *build* in Travis CI is a sequence of [stages](../for-beginners/#builds-jobs-stages-and-phases). Each *stage* consists of [jobs](../for-beginners/#builds-jobs-stages-and-phases) run in parallel. 
```

Comment:
Drop the parens. The parent that splits one sentence and continues into another sentence really breaks flow.

Also, the statement in parens are like whispers. I don't think you should whisper what a build is. You should clearly state it.  In contrast, dashes - like this - is like yelling. Dashes are intended to make something really stand out. In general, don't whisper and don't yell. Write in short sentences like you would talk.